pip/basecommand.py

```diff
@@ -214,10 +215,20 @@ def main(self, args):
         if options.require_venv:
             # If a venv is required check if it can really be found
             if not running_under_virtualenv():
-                logger.critical(
-                    'Could not find an activated virtualenv (required).'
-                )
-                sys.exit(VIRTUALENV_NOT_FOUND)
+                if not getattr(options, 'use_user_site'):
+                    logger.critical(
+                        textwrap.dedent("""\
+                        ******************************************************
+                        Using pip outside a virtualenv and without --user
+                        is not recommended!
```

Comment:
don't think this is the place to log a  message about best practices.

if they've set the option, and not in a virtualenv, then just log the facts like it is now.